frog/imports/client/TeacherView/OrchestrationView.jsx

```diff
@@ -82,7 +82,7 @@ class OrchestrationViewController extends React.Component<any, {}> {
           <Grid item xs={12} className={classes.maybeScaled}>
             <Grid container className={classes.subroot}>
               <Grid item xs={12}>
-                {visible ? (
+                {visible || session.singleActivity ? (
```

Comment:
Because session is of type `any`, we have no guarantee that singleActivity would exist. If within the scope of this PR, we could add a prop type for this component specifying the type of the session object

Reply:
In that case, `undefined` would still be false-y and the logic should work fine.
I think no change is required here.